src/third_party/histogram/histogram.hpp

```diff
@@ -75,13 +75,15 @@ class Histogram
   {
     if( x < Start )
       ++underflow;
+    else if( x > End )
+      ++overflow;
     else
     {
       const size_t i(
         static_cast<size_t>(
         (x-Start)*nBins_by_interval) );
-      if( i < nBins ) ++freq[i];
-      else ++overflow;
+      // clamp for the particular case when (x == End)
```

Comment:
If it's an overflow the data is not contained in the range of the histogram.
I mean if you want a histogram for the range [0;10] and if a data is 15, it does not fit in the domain of definition.
So you change here the behavior of the class!